sdk/core/Azure.Core/samples/DynamicContent.md

```diff
@@ -0,0 +1,73 @@
+# Azure SDK Dynamic JSON samples
+
+Azure SDK client [Protocol methods](ProtocolMethods.md) do not take or return model types.  JSON response content can be accessed using BCL JSON types, but use of these APIs can result in code that is difficult to read and obscures the author's intent.  To improve the developer experience, Azure.Core provides a dynamic layer over JSON APIs.
+
+## Accessing Response Content
+
+### Get `dynamic` from response
+
+Dynamic content is obtained from the `Response` return value.
+
+```C# Snippet:GetDynamicJson
+Response response = await client.GetWidgetAsync("123");
+dynamic widget = response.Content.ToDynamic();
+```
+
+### Get a JSON property
+
+JSON properties are read using dynamic member access.
+
+```C# Snippet:GetDynamicJsonProperty
+Response response = await client.GetWidgetAsync("123");
+dynamic widget = response.Content.ToDynamic();
+string name = widget.Name;
```

Comment:
At least in my work for CLU, the default seems to be originally-cased (should be camel, based on API guidelines) property accessors.

Reply:
It would be good to explain to users how they can know what properties they can access, i.e. REST API docs and debugger.

Reply:
> At least in my work for CLU, the default seems to be originally-cased (should be camel, based on API guidelines) property accessors.

Updated to show intended usage via defaults (and will update again once https://github.com/Azure/azure-sdk-for-net/pull/35777 lands)

Reply:
Added a section on REST API docs and the VS Dynamic View.